Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -802,8 +802,8 @@
                 "filterClause": "AND",
                 "filters": [
                     {"operator": "IS_NOT", "term": "exposure", "value": "IsPublic"},
-                    {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
                     {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
+                    {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
```

Comment:
Why was this change necessary?

Reply:
Look like auto-formatting ordering the dict values? I am not sure. This change is already in the base branch and I don't think I made it. I just want the readme changes in, and we already have a PR open, so that is why I targeted that branch